src/algorithms/search/BinarySearch.java

```diff
@@ -0,0 +1,56 @@
+package algorithms.search;
+
+import java.util.Arrays;
+
+public class BinarySearch {
+
+
+    /**
+     * This function processes the search binary algorithm using a while loop
+     * it return the index of the element if it found, otherwise it returns -1
+     * @param nums
+     * @param target
+     * @return
+     */
+    public static int find(int [] nums,int target){
+        int left = 0;
```

Comment:
Hello @FimboIsso, great job so far.

I noticed the usage of static methods in some parts of the code, and I'd like to suggest considering an alternative approach to align with our project structure. Using static methods can sometimes hinder testability and make code harder to maintain.

I suggest` Instance Methods` instead of `static` ones